src/__tests__/config.test.ts

```diff
@@ -112,6 +119,38 @@ describe(Config, () => {
     expect(readFileSync("appmap.yml").toString()).toEqual(malformedAppMapFileContent);
   });
 
+  describe("migrate", () => {
+    it("sets the language field in existing configs", () => {
+      writeFileSync(
+        "appmap.yml",
+        YAML.stringify({
+          name: "test-package",
+          appmap_dir: "appmap",
+          packages: [{ path: "." }],
```

Comment:
Can we add a test of migration with user comments, to check they're preserved?

Reply:
I would say skip this. Let's get this out the door.